File: src/apps/Andriod/app/src/main/java/com/dkellycollins/triggerfinger/util/logger/ILogger.java
package com.dkellycollins.triggerfinger.util.logger;

/**
 * Created by Devin on 8/22/2015.
 */
public interface ILogger {

    void debug(String message);

    void debug(String message, Throwable exception);

    void info(String message);

    void warn(String message);

    void warn(String message, Throwable exception);

    void error(String message, Throwable exception);

}

File: src/apps/Android-Kotlin/app/src/main/kotlin/com/dkellycollins/triggerfinger/util/logger/ILogger.kt
package com.dkellycollins.triggerfinger.util.logger

/**
 * Created by Devin on 8/22/2015.
 */
interface ILogger {

    fun debug(message: String)

    fun debug(message: String, exception: Throwable)

    fun info(message: String)

    fun warn(message: String)

    fun warn(message: String, exception: Throwable)

    fun error(message: String, exception: Throwable)

}
